### DISCUSSION TITLE
Phase Field Fracture model with shell elements?

### DISCUSSION THREAD

**Discussion by MartimSalgado** [idaholab/moose#17537] [Q&A Modules: Solid mechanics] [answered]
I am modeling the fracture of a long fiber reinforced composite material and would like to use shell elements (each layer is one shell element).
As anything like that been done with MOOSE? I've been searching through the examples in "combined" for TesnorMechnics +PhaseField stuff but haven't found anything.
It is also unclear to me how we can choose either plane tress or plane strain, or other types of elements.

Thanks for the assistance!
Martim

**Comment by jiangwen84** [comment, marked as answer]
We have not implemented phase field fracture model for shell elements.  As for now,  maybe you have to consider to use solid elements.
